server/__init__.py

```diff
@@ -108,15 +109,20 @@ async def create_syncsketch_webhook(self):
             "Content-Type": "application/json",
         }
 
-        existing_webhooks = requests.request(
-            "GET", syncsketch_endpoint, headers=headers)
-
-        if existing_webhooks.status_code == 400:
-            logging.error(existing_webhooks.json())
+        try:
+            async with httpx.AsyncClient(timeout=timeout) as client:
+                res = await client.get(
+                    syncsketch_endpoint,
+                    headers=headers,
+                )
+                res.raise_for_status()
+                existing_webhooks = res.json()
+        except Exception:
+            log_traceback("Unable to get existing Webhooks in SyncSketch.")
```

Comment:
Does this include the parent stack trace from the exception in the log? :) If so, nice!

Reply:
yep
![image](https://github.com/user-attachments/assets/c2dc8136-d63f-40be-955c-339113d93c92)